packages/connection-tag-utils/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@interledger/connection-tag-utils",
+  "version": "0.0.0",
+  "author": "Interledger Team <info@interledger.org>",
+  "license": "Apache-2.0",
+  "description": "Utilities for encrypted STREAM connection tags",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*.js",
+    "dist/**/*.js.map",
+    "dist/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rimraf .nyc_output/* dist/* coverage/*",
+    "build": "tsc -p tsconfig.build.json",
+    "test": "mocha --require ts-node/register/transpile-only --ui mocha-typescript --require source-map-support/register --watch-extensions ts test/*.ts",
+    "cover": "nyc --extension .ts --reporter=json --reporter=html --require ts-node/register/transpile-only --ui mocha-typescript --require source-map-support/register mocha test/*.spec.ts",
+    "codecov": "codecov --root=../../ -f coverage/*.json -F connection-tag-utils"
```

Comment:
@sharafian The flag name should contain underscores. It has to match the regex ^[a-z0-9_]{1,45}$